bayesian/utility/v2/component_analyses.py

```diff
@@ -168,6 +168,50 @@ def get_batch_ca_data(ecosystem: str, packages: List) -> dict:
     return response
 
 
+def get_batch_ca_vulnerability_data(ecosystem: str, packages: List) -> dict:
+    """Fetch package details for component analyses."""
+    logger.debug('Executing get_batch_ca_data')
+    started_at = time.time()
+
+    response = None
+    semver_packages = []
+    pseudo_version_packages = []
+
+    # Need to seperate semver and pseudo verion packages for golang
+    if (ecosystem == "golang"):
```

Comment:
for the time being, lets keep it simple. we dont need to support golang or semver etc as i dont think its yet supported in clair / quay. when we get there, we can update the code here.

Reply:
Yes.I agree.I had kept it just in case.